M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -612,6 +649,19 @@ RnReductionP(ZZ, Ring, ZZ):= opts -> (n1, R1, p)-> (
     return Rn(n1, ambRing/Id, opts);
 );
 
+verifyStrategy := (passedStrat) -> (
+    if not ((class passedStrat === HashTable) or ((class passedStrat) === MutableHashTable)) then return false;
+    if not (passedStrat #? LexLargest) then return false;
```

Comment:
Rewrite using the logical operator `and`, without `if` or `return`.

Reply:
I believe I've done this.